posthog/models/plugin.py

```diff
@@ -0,0 +1,23 @@
+from django.contrib.postgres.fields import JSONField
+from django.db import models
+
+
+class Plugin(models.Model):
+    name: models.CharField = models.CharField(max_length=200, null=True, blank=True)
+    description: models.TextField = models.TextField(null=True, blank=True)
+    url: models.CharField = models.CharField(max_length=800, null=True, blank=True)
```

Comment:
There's a [URLField](https://docs.djangoproject.com/en/3.1/ref/models/fields/#urlfield) that already includes certain validations (particularly because these will be propagated to the serializer.

Reply:
fixed and it seems to work with "file:" urls as well